<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>MainWindow</class>
  <widget class="QMainWindow" name="MainWindow">

    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>484</width>
        <height>458</height>
      </rect>
    </property>

    <property name="windowTitle">
      <string>Calculator</string>
    </property>

    <property name="styleSheet">
      <string notr="true">background-color: black;</string>
    </property>

    <widget class="QWidget" name="centralWidget">

      <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>

      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QTextBrowser" name="result">
                
            <property name="font">
              <font>
                <pointsize>50</pointsize>
                <weight>75</weight>
                <bold>true</bold>
              </font>
            </property>

            <property name="styleSheet">
              <string notr="true">color: white;</string>
            </property>

            <property name="placeholderText">
              <string>0</string>
            </property>

          </widget>
        </item>

        <item>
          <layout class="QGridLayout" name="gridLayout">

            <item row="3" column="2">
              <widget class="QPushButton" name="ButtonNumberThree">

                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>3</string>
                </property>

                <property name="shortcut">
                  <string>3</string>
                </property>

                <property name="autoDefault">
                  <bool>false</bool>
                </property>

              </widget>
            </item>

            <item row="3" column="3">
              <widget class="QPushButton" name="ButtonSub">
                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: orange;color: white;</string>
                </property>

                <property name="text">
                  <string>-</string>
                </property>

                <property name="shortcut">
                  <string>-</string>
                </property>

              </widget>
            </item>

            <item row="3" column="1">
              <widget class="QPushButton" name="ButtonNumberTwo">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>2</string>
                </property>

                <property name="shortcut">
                  <string>2</string>
                </property>

              </widget>
            </item>

            <item row="1" column="1">
              <widget class="QPushButton" name="ButtonNumberEight">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>8</string>
                </property>

                <property name="shortcut">
                  <string>8</string>
                </property>

              </widget>
            </item>

            <item row="2" column="1">
              <widget class="QPushButton" name="ButtonNumberFive">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>5</string>
                </property>

                <property name="shortcut">
                  <string>5</string>
                </property>

              </widget>
            </item>

            <item row="2" column="2">
              <widget class="QPushButton" name="ButtonNumberSix">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>6</string>
                </property>

                <property name="shortcut">
                  <string>6</string>
                </property>

              </widget>
            </item>

            <item row="1" column="3">
              <widget class="QPushButton" name="ButtonDiv">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: orange;color: white;</string>
                </property>

                <property name="text">
                  <string>รท</string>
                </property>

                <property name="shortcut">
                  <string>/</string>
                </property>

              </widget>
            </item>

            <item row="1" column="0">
              <widget class="QPushButton" name="ButtonNumberSeven">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>7</string>
                </property>

                <property name="shortcut">
                  <string>7</string>
                </property>

              </widget>
            </item>

            <item row="3" column="0">
              <widget class="QPushButton" name="ButtonNumberOne">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>1</string>
                </property>

                <property name="shortcut">
                  <string>1</string>
                </property>

              </widget>
            </item>

            <item row="1" column="2">
              <widget class="QPushButton" name="ButtonNumberNine">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>9</string>
                </property>

                <property name="shortcut">
                  <string>9</string>
                </property>

              </widget>
            </item>

            <item row="8" column="3">
              <widget class="QPushButton" name="ButtonAdd">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: orange;color: white;</string>
                </property>

                <property name="text">
                  <string>+</string>
                </property>

                <property name="shortcut">
                  <string>+</string>
                </property>

              </widget>
            </item>

            <item row="2" column="3">
              <widget class="QPushButton" name="ButtonMul">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: orange;color: white;</string>
                </property>

                <property name="text">
                  <string>x</string>
                </property>

                <property name="shortcut">
                  <string>*</string>
                </property>

              </widget>
            </item>

            <item row="8" column="0" colspan="3">
              <widget class="QPushButton" name="ButtonNumberZero">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                 <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>0</string>
                </property>

                <property name="shortcut">
                  <string>0</string>
                </property>

              </widget>
            </item>

            <item row="2" column="0">
              <widget class="QPushButton" name="ButtonNumberFour">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: white;</string>
                </property>

                <property name="text">
                  <string>4</string>
                </property>

                <property name="shortcut">
                  <string>4</string>
                </property>

              </widget>
            </item>

            <item row="10" column="0">
              <widget class="QPushButton" name="ButtonClear">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>20</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: grey;color: white;</string>
                </property>

                <property name="text">
                  <string>C</string>
                </property>

                <property name="shortcut">
                  <string>C</string>
                </property>

              </widget>
            </item>

            <item row="10" column="1" colspan="3">
              <widget class="QPushButton" name="ButtonEqual">

                <property name="minimumSize">
                  <size>
                    <width>0</width>
                    <height>50</height>
                  </size>
                </property>

                <property name="font">
                  <font>
                    <pointsize>27</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                  </font>
                </property>

                <property name="styleSheet">
                  <string notr="true">background-color: orange;color: white;</string>
                </property>

                <property name="text">
                  <string>=</string>
                </property>

                <property name="shortcut">
                  <string>Return</string>
                </property>

              </widget>
            </item>

          </layout>
        </item>

      </layout>
    </widget>

  <widget class="QStatusBar" name="statusBar"/>
    <action name="actionExit">
      <property name="text">
        <string>Exit</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+Q</string>
      </property>
    </action>
    <action name="actionReset">
      <property name="text">
        <string>Reset</string>
      </property>
      <property name="shortcut">
        <string>Ctrl+R</string>
      </property>
    </action>
  </widget>

  <layoutdefault spacing="6" margin="11"/>
  
  <resources/>

  <connections/>

</ui>
